### PR TITLE
Add-resources

### PR DESCRIPTION
**Pull request by almajd3713**
add persisting reading of times the program was launched. Some fixes to accomodate were included as well.

### CODE REVIEW

File: src/com/softengdev/PersistantData.java
Comment (Abd-Elhakim-Arabet) [inline src/com/softengdev/PersistantData.java:1]: 1. Potential solutions for the file not found exception:
  -log the error, or just display a message indicating the file not found
  -create the file
  
2. Fix spelling : Change _PersistantData_ to _PersistentData_
to stay persistent :)

3. Possible error:
  in the load_value_int() m  in the load_value_int() method the value inside the file might be a type different than integer, handle this exception method the value inside the file might be a type different than integer, handle this exception

**Review by Abd-Elhakim-Arabet — CHANGES_REQUESTED** [review]
Potential solutions for the file not found exception:
-log the error, or just display a message indicating the file not found
-create the file

Fix spelling : Change PersistantData to PersistentData
to stay persistent :)

Possible error:
in the load_value_int() m in the load_value_int() method the value inside the file might be a type different than integer, handle this exception method the value inside the file might be a type different than integer, handle this exception

**Comment by almajd3713** [comment]
> Possible error: in the load_value_int() m in the load_value_int() method the value inside the file might be a type different than integer, handle this exception method the value inside the file might be a type different than integer, handle this exception

load_value_string is supposed to handle strings, which is the default value received from files. load_value_int just turns it into an integer, and vice versa. This is intended behavior

> Potential solutions for the file not found exception:

The exception will be needed only if the file may not exist. As it comes already with the build version, this isn't really necessary. Needless to say, it will be handled

**Review by Abd-Elhakim-Arabet — APPROVED** [review]
Seems good for me

**Pull request merged**